pkg/ambex/main.go

```diff
@@ -137,6 +137,10 @@ type Args struct {
 
 	snapdirPath string
 	numsnaps    int
+
+	// edsByPass will bypass using EDS and will insert the endpoints into the custer data manually
```

Comment:
spelling: `custer` -> `cluster` no reason to do an extra CI run for it but if you do make other changes.